src/interop/lionweb.ts

```diff
@@ -77,27 +77,39 @@ export class LanguageMapping {
     }
 }
 
-function isASTNode(classifier: Classifier) {
-    return (classifier instanceof Concept) &&
-        (classifier.key == ASTNode.key || (classifier.extends && isASTNode(classifier.extends)));
+function isSpecialConcept(classifier: Classifier) {
+    return classifier.key == PositionClassifier.key;
 }
 
 function importFeature(feature: LWFeature): Feature | undefined {
     const def: Feature = { name: feature.name };
     if (feature instanceof Containment) {
-        if (feature.type && isASTNode(feature.type)) {
+        if (feature.type && feature.type instanceof Concept && !isSpecialConcept(feature.type)) {
             def.child = true;
             def.multiple = feature.multiple;
         } else {
-            // TODO we assume that:
-            //  1) we're importing a StarLasu AST
-            //  2) containments in a StarLasu AST are either AST nodes or internal StarLasu objects like the position
+            // TODO is it possible in Lionweb to have a Containment that's not a concept?
```

Comment:
Yes: it could be a ConceptInterface

Reply:
Ok, it makes sense to support that as well.